edgePy/DGEList.py

```diff
@@ -98,16 +109,31 @@ def counts(self, counts: np.ndarray) -> np.matrix:
             counts: Columns correspond to samples and row to genes.
 
         """
+        if counts is None:
+            self._counts = None
+            return
```

Comment:
Can del lines 112-114 as the property for counts is defaulted to `None` already in the `__init__()`.